jest.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   setupFilesAfterEnv: ['./jest.setup.js'],
   modulePathIgnorePatterns: ['<rootDir>/mashup-example'],
   transformIgnorePatterns: ['<rootDir>/node_modules/(?!(@qlik-trial)/)'],
+  coveragePathIgnorePatterns: ['src/__test__/generate-test-data.ts', 'src/__test__/test-with-providers.tsx']
```

Comment:
We should move the test utils files to some other location but I'll do that in another PR as I dont want all those changed test files in this PR as well